common/Utils.hpp

```diff
@@ -13,6 +13,13 @@
 #include <typeinfo>
 #include <vector>
 #include <QtWidgets>
+#include <memory>
+namespace std {
```

Comment:
Hey, can we avoid putting things in the `std` namespace?  We can have our own `make_unique` until we get c++14 working, but it's kinda bad practice to put things into `std`.